ic3ia/utils.cpp

```diff
@@ -82,11 +82,11 @@ msat_config get_config(ModelGeneration model, bool interpolation)
         // interpolation for BV requires the lazy solver
         OPT_("theory.bv.bit_blast_mode", "0");
         OPT_("theory.bv.eager", "false");
-	// This option is a hack to handle interpolation failure in the
-	// case of QF_UFLIA
-	OPT_("theory.la.interpolation_laz_split_mixed_eq", "false");
+        // This option is a hack to handle interpolation failure in the
+        // case of QF_UFLIA
+        OPT_("theory.la.interpolation_laz_split_mixed_eq", "false");
```

Comment:
let's remove this option

Reply:
Just for future reference: we decided offline to keep this option for now and change it in a separate PR